README.md

```diff
@@ -7,367 +7,506 @@
+be the list of device attributes that will be considered lazy (or passive). With
+this information, the IoT Agent sends a new `registerContext` request to the
+Context Broker, registering itself as ContextProvider of all the lazy attributes
+for the device's entity. The `registrationId` is then stored along the other
+device information inside the IoT Agent device registry.
+
+As NGSI9 does not allow the context registrations to be removed, when the device
+is removed from the IoT Agent, the registration is updated to an expiration date
+of 1s, so it is effectively disabled. Once it has been disabled, the device is
+removed from the IoT Agent's internal registry.
 
 #### Lazy attributes
-When a request for data from a lazy attribute arrives to the Context Broker, it forwards the request to the Context
-Provider of that entity, in this case the IoT Agent. The IoT Agent will in turn ask the device for the information needed,
-transform that information to a NSGI format and return it to the Context Broker. The latter will the forward the response
-to the caller, transparently.
+
+When a request for data from a lazy attribute arrives to the Context Broker, it
+forwards the request to the Context Provider of that entity, in this case the
+IoT Agent. The IoT Agent will in turn ask the device for the information needed,
+transform that information to a NSGI format and return it to the Context Broker.
+The latter will the forward the response to the caller, transparently.
 
 #### Commands
 
-**IMPORTANT NOTE:** at the present moment, commands (both push and poll) are supported only in the case of explictely
-provisioned agents. For autoprovisioned agents commands are not currently supported, although
-[an issue](https://github.com/telefonicaid/iotagent-node-lib/issues/572) has been created about this functionality.
+**IMPORTANT NOTE:** at the present moment, commands (both push and poll) are
+supported only in the case of explictely provisioned agents. For autoprovisioned
+agents commands are not currently supported, although
+[an issue](https://github.com/telefonicaid/iotagent-node-lib/issues/572) has
+been created about this functionality.
 
-Commands are modelled as updates over a lazy attribute. As in the case of the lazy attributes, updates over a command
-will be forwarded by the Context Broker to the IoT Agent, that will in turn interact with the device to perform the
-requested action. Parameters for the command will be passed inside the command value.
+Commands are modelled as updates over a lazy attribute. As in the case of the
+lazy attributes, updates over a command will be forwarded by the Context Broker
+to the IoT Agent, that will in turn interact with the device to perform the
+requested action. Parameters for the command will be passed inside the command
+value.
 
 There are two differences with the lazy attributes:
-* First of all, for every command defined in a device, two new attributes are created in the entity with the same name
```

Comment:
This part seems has broken.

In master:

![imagen](https://user-images.githubusercontent.com/1534240/48704862-ee2a5000-ebf7-11e8-89d1-0c279a5517ab.png)


In this PR:

![imagen](https://user-images.githubusercontent.com/1534240/48704847-e8cd0580-ebf7-11e8-8ffe-9c7e03655016.png)

Reply:
Fixed - e12947e